addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4791,19 +4799,26 @@ registry['sizing_x'] = registry.sizing.extend({
                     colSize = 1;
                     offset = beginOffset + beginCol - 1;
                 }
-                this.$target.attr('class', this.$target.attr('class').replace(/\s*(offset-xl-|offset-lg-|col-lg-)([0-9-]+)/g, ''));
+                const offsetColRegex = new RegExp(`\\b(offset-${resolutionModifierWithXL}|col-${resolutionModifier})(\\d{1,2})`, 'g');
+                this.$target.attr('class', this.$target.attr('class').replace(offsetColRegex, ''));
```

Comment:
We try to get rid of JQuery, you can take the opportunity to replace it by a pure JS code :+1: